frontend/src/member/MemberModal.tsx

```diff
@@ -1,9 +1,126 @@
-import React from 'react';
+import {MemberModel} from "./MemberModel";
+import {ChangeEvent, FormEvent, useState} from "react";
+import axios from "axios";
+import styled from "styled-components";
+
+type ModalProps = {
+    closeModal: () => void
+    member: MemberModel
+    fetchAllMembers: () => void
+}
+
+export default function MemberModal(props: ModalProps) {
+
+    const [newFirstName, setFirstName] = useState("")
+    const [newLastName, setLastName] = useState("")
+    const [newStreet, setStreet] = useState("")
+    const [newZipcode, setZipcode] = useState("")
+    const [newCity, setCity] = useState("")
+    const [newEmail, setEmail] = useState("")
```

Comment:
Could be shortened by using a useState that takes a whole member-object.
Would also make the all the handleXyz obsolete. You can have a look here: https://github.com/ChristianBorn/Capstone/blob/master/frontend/src/stock/EditItemModal.tsx